packages/client/src/views/Settings/SettingsPage.tsx

```diff
@@ -48,88 +48,22 @@ import { RouteComponentProps, StaticContext } from 'react-router'
 import { SETTINGS } from '@client/navigation/routes'
 import { Navigation } from '@client/components/interface/Navigation'
 import { AvatarChangeModal } from './AvatarChangeModal'
-import { ImageLoader } from './ImageLoader'
-import { IImage } from '@client/utils/imageUtils'
+import { IImage, validateImage, ERROR_TYPES } from '@client/utils/imageUtils'
+import { Content } from '@opencrvs/components/lib/interface/Content'
+import { ALLOWED_IMAGE_TYPE } from '@client/utils/constants'
 
-const Container = styled.div`
-  ${({ theme }) => theme.shadows.light};
-  color: ${({ theme }) => theme.colors.copy};
-  background: ${({ theme }) => theme.colors.white};
-  padding: 40px 77px;
-  margin: 36px auto;
-  width: 1140px;
-  @media (max-width: ${({ theme }) => theme.grid.breakpoints.lg}px) {
-    margin: 0;
-    padding: 24px 0;
-    width: 100%;
-    min-height: 100vh;
-    margin-top: 0;
-    box-shadow: 0 0 0 rgba(0, 0, 0, 0);
-  }
+const HiddenInput = styled.input`
+  display: none;
 `
 
 const BodyContainer = styled.div`
+  margin-top: 48px;
   margin-left: 0px;
   @media (min-width: ${({ theme }) => theme.grid.breakpoints.lg}px) {
     margin-left: 249px;
   }
```

Comment:
I think the `BodyContainer` shouldn't be here. It might break the common design and will add extra maintenance. This should be part of the `Content` component.

Reply:
Should I just skip this body container?

Reply:
Yes I agree. The Content component should set the padding for the bodyContent. and with the ListView handle the responsiveness/mobile views